e2e/specs/st_button.spec.js

```diff
@@ -14,6 +14,20 @@
  * limitations under the License.
  */
 
+const NO_OF_BUTTONS = 13
+/**
+ * Order of CONN_TYPES_BUTTONs matters as they are rendered in order in st_button.py
+ */
+const CONN_TYPES_BUTTONS = [
+    "snowflake",
+    "aws_s3",
+    "gsheets",
+    "bigquery",
+    "http_file",
+    "custom",
+    "huggingface",
+    "postgresql"]
```

Comment:
super nit (I'm surprised `precommit` didn't pick this up): closing `]` should be on the next line

Reply:
it's updated 👍